number/src/traits.rs

```diff
@@ -149,6 +149,13 @@ pub trait FieldElement:
     fn try_into_i32(&self) -> Option<i32>;
 }
 
+// TODO: should we bother with this or just go through bignumber?
```

Comment:
I imagine big number would be fine

Reply:
I suspect encoding it in the type system as I did would cleanly solve [this](https://github.com/powdr-labs/powdr/blob/7aca9ecbd341471247e719b724994a01983677ca/halo2/src/prover.rs#L43) as you can express the fact that this works for any field which has a halo2/plonky3 counterpart